environments/aks/perftest.tfvars

```diff
@@ -3,6 +3,7 @@ kubernetes_cluster_version         = "1.19.11"
 kubernetes_cluster_agent_min_count = "55"
 kubernetes_cluster_agent_max_count = "60"
 kubernetes_cluster_ssh_key         = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC2lguwg1h0qcaPqPZutQChBAtDK9USDTKNpnY3miVD0cwtFE/Q8U9A3KAfR0wI/WOystKXKGO8e3wq8xf6Pe08aCrbi7X8zIsixKgpQiNXT3j1zRz2Ae4Sa06znSiyzadCv4gSzWsq6m7Sq3FQJ7f2/USDemm1yA0Nena8g73IjxFe0zErqtnRhzicaccxDxaoZNBrfRotV+Nz6FEegUkVnqr+5Jy4H3XvdXfDPc1UzDAn0iBptEW80tcyKZsj7l2Cl20JjnSZ2PwGX/FMzzZIeTtR+eo7/3HxiaZumYAFLcfQ+ZCzId5hA6g30hsSi17HlMco//KkfgReaxXz+gDT aks-ssh"
+enable_user_system_nodepool_split  = true
 
 system_node_pool = {
```

Comment:
I dont think these values are correct?
https://github.com/hmcts/aks-cft-deploy/blob/DTSPO-5558-add-user-and-system-pools-perftest/environments/aks/perftest.tfvars#L8-L15